exiftool.go

```diff
@@ -64,6 +67,11 @@ func NewExiftool(opts ...func(*Exiftool) error) (*Exiftool, error) {
 	return &e, nil
 }
 
+// SetBufferSize changes the buffer size
+func (e *Exiftool) SetBufferSize(size int) {
```

Comment:
I think that the buffer size specification should rather be specified as a [functionnal options](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis).

The thing that bothers me with this implementation (a method on the `ExifTool` struct) is that users could think that the buffer size could be set **after** `ExifTool` initiatialization, but it doesn't.

Could you please change your implementation to fit a functionnal option (`func WithBufferSize func(*Exiftool) error { ... }`) ? The `NewExifTool` function already accepts functionnal options.

Reply:
I don't mention unit-tests and the update of the documentation :)